### PR TITLE
Bump version number to prevent tar.Extract error.

### LINKED ISSUE #42: TypeError: tar.Extract is not a function
```
.../theme-manager/node_modules/tarball-extract/tarball.js:11
    .pipe(tar.Extract({ path: destination}))
              ^

TypeError: tar.Extract is not a function
```

I needed to bump `tarball-extract` version from 0.0.3 to 0.0.5 or higher.

### PR DESCRIPTION
**Pull request by whizkid77**
Fixes #42

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
+#emacs
+*~
+
+# JetBrains IDE stuff
+.idea
+
```
Comment (PeterDaveHello) [inline .gitignore]: Would you please remove the change of `.gitignore` as it's not part of the issue we're going to solve here? Thanks.

**Comment by PeterDaveHello** [comment]
@whizkid77 would you like to get it done? Thanks.

**Comment by whizkid77** [comment]
Ah sorry, I'll close this one and open a clean PR later.

**Comment by PeterDaveHello** [comment]
Thanks a lot!

**Pull request closed without merging**